From e4aca10e37c66357f17aea241431d01f095495c3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rafael=20Laboissi=C3=A8re?= <rafael@laboissiere.net>
Date: Sun, 7 Mar 2021 17:14:28 +0100
Subject: [PATCH] Include <limits>

This avoids compilation errors related to the non existent
std::numeric_limits in g++ 11.

It also works with g++ 10.
---
 melder/melder.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/melder/melder.h b/melder/melder.h
index 7ff94a336..9e40751b2 100644
--- a/melder/melder.h
+++ b/melder/melder.h
@@ -33,6 +33,7 @@
 #include <memory>   // unique_ptr
 #include <new>   // placement new
 #include <algorithm>   // std::min
+#include <limits>   // std::numeric_limits
 
 /*
 	Law of Demeter for class functions defined outside class definition.
